velox/functions/lib/QuantileDigest.h

```diff
@@ -109,6 +109,16 @@ class QuantileDigest {
           CdfEntry>>
   getDistributionFunction(double rangeStart, double rangeEnd) const;
 
+  // Calculate the truncated mean between quantile bounds
+  // Returns null when empty
```

Comment:
Hi @natashasehgal, could you add more explanation in the comment about what "truncated mean" is? Maybe add a small example if that would help understand it.